Chat_App/Chat_App/UI/TypingComponentView.swift

```diff
@@ -0,0 +1,188 @@
+//
+//  TypinComponentView.swift
+//  Chat_App
+//
+//  Created by Gi Oo on 26.04.23.
+//
+
+import UIKit
+
+class TypingComponentView: UIView {
+
+    // MARK: - Properties
+    private var heightConstraint: NSLayoutConstraint? = nil
+
+    private lazy var containerView: UIView = {
+        let view = UIView()
+        view.layer.borderWidth = Constants.containerBorderWidth
+        view.layer.borderColor = AppColors.containerBorderColor
+        view.layer.cornerRadius = Constants.containerCornerRadius
+        view.translatesAutoresizingMaskIntoConstraints = false
+        return view
+    }()
+
+    private lazy var textView: UITextView = {
+        let textView = UITextView()
+        textView.text = Constants.placeholderText
+        textView.translatesAutoresizingMaskIntoConstraints = false
+        textView.isScrollEnabled = false
+        textView.textColor = .lightGray
+        textView.font = .systemFont(ofSize: Constants.textViewFontSize)
+        textView.delegate = self
+        textView.backgroundColor = UIColor.clear
+        return textView
+    }()
+
+    private lazy var button: UIButton = {
+        let button = UIButton()
+        button.setImage(UIImage(named: Constants.sendButtonImageName), for: .normal)
+        button.addTarget(self, action: #selector(buttonTapped), for: .touchUpInside)
+        button.translatesAutoresizingMaskIntoConstraints = false
+        return button
+    }()
+
+    // MARK: - Initializers
+    init() {
+        super.init(frame: .zero)
+        updateTextViewHeight()
```

Comment:
```suggestion
        setupTextViewHeight()
```

It'd be better to use `setup` because you initialize your view here